js/navigation/base.js

```diff
@@ -10,60 +10,64 @@ define([
 
 (function( $, undefined ) {
 
-	// existing base tag?
-	var baseElement = $( "head" ).children( "base" ),
+	var base,
+
+		// existing base tag?
+		baseElement = $( "head" ).children( "base" ),
+
+		// DEPRECATED as of 1.5.0 and will be removed in 1.6.0. As of 1.6.0 only
+		// base.dynamicBaseEnabled will be checked
+		getDynamicEnabled = function() {
+			if ( $.mobile.dynamicBaseEnabled !== undefined ) {
+				return $.mobile.dynamicBaseEnabled;
+			}
+			return base.dynamicBaseEnabled;
+		};
 
 	// base element management, defined depending on dynamic base tag support
 	// TODO move to external widget
 	base = {
 
-		// define base element, for use in routing asset urls that are referenced
+		// disable the alteration of the dynamic base tag or links
+		dynamicBaseEnabled: true,
+
+		// Make sure base element is defined, for use in routing asset urls that are referenced
 		// in Ajax-requested markup
-		element: ( baseElement.length ? baseElement :
-			$( "<base>", { href: $.mobile.path.documentBase.hrefNoHash } ).prependTo( $( "head" ) ) ),
+		element: function() {
+			if ( !baseElement || baseElement.length === 0 ) {
+				baseElement = $( "<base>", { href: $.mobile.path.documentBase.hrefNoSearch } )
```

Comment:
Note that the new code initializes the newly generated base tag with ```hrefNoSearch``` instead of ```hrefNoHash```, because ```reset()``` below uses ```hrefNoSearch``` for the reset value and because the query string does not belong in the base URL.